server/service/src/document/patient/query.rs

```diff
@@ -140,3 +140,52 @@ impl PatientFilter {
         }
     }
 }
+
+impl PatientFilter {
```

Comment:
as per previous pr i think this should just be name filter

Reply:
I made a note